tests/test_legal.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+
+from webservices.resources.legal import parse_query_string
+
+class LegalPhraseSearchTests(unittest.TestCase):
```

Comment:
Can you add test cases for `"phrase only"` and `"phrase followed by" term`.  Regexes do weird things sometimes with string starts.